### ISSUE TITLE
移除WETV

### ISSUE THREAD

**Issue by malagebidi** [felixonmars/dnsmasq-china-list#504]
腾讯视频海外版的两个域名，目前wetv已禁止国内访问，建议移除

> wetv.vip
> wetvinfo.com

**Comment by fernvenue** [comment]
Seems we should keep `wetvinfo.com`:

```
~$ dig NS wetvinfo.com +short
ns3.dnsv5.com.
ns4.dnsv5.com.
```

Just a quick clarification, this repository is intended for DNS purposes not proxy services :)

**Comment by felixonmars** [comment]
```
wetv.vip.               172800  IN      NS      ns-open1.qq.com.
wetv.vip.               172800  IN      NS      ns-open2.qq.com.
wetv.vip.               172800  IN      NS      ns-open3.qq.com.
```

And it still resolves to China Mainland IP with 8.8.8.8 in Hong Kong.
